src/main/configurations/Translate/Common/xsl/CreateIdentication.xslt

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+    <xsl:param name="zaak-identificatie-template" />
+    <xsl:output method="xml" indent="yes" omit-xml-declaration="yes" />
+    <xsl:template match="/">
+        <xsl:variable name="test"><xsl:value-of
+                select="$zaak-identificatie-template" /></xsl:variable>
+    <xsl:variable
+            name="patternArray" select="tokenize($test,'-')" />
+            <xsl:variable
+            name="identificatie" select="/result/rowset/row/field" />
+            <xsl:for-each
+            select="$patternArray">
+            <xsl:choose>
+                <xsl:when test="contains(current(), 'datetime')">
+                    <xsl:value-of select="substring-before(current(),'{')" />
+                        <xsl:variable
+                        name="date"
+                        select="substring-before(substring-after(current(),':'), '}')" />
+                        <xsl:value-of
+                        select="concat(format-dateTime(current-dateTime(), $date),'-')" />
```

Comment:
Is missing a check for the optional ':', indicating that a format string follow. Without ':' it should just return current-dateTime without any formatting.